### ISSUE TITLE
New album titles are called "Surname"

### ISSUE THREAD

**Issue by RikkiBC** [pulsejet/memories#1244]
### Describe the bug

When going to create a new album, the field for the album title is called "Surname".

### Steps To Reproduce

1. Go to add photos to an album.
2. Create a new album to add them to.

### Platform

```markdown
- OS: Android and Web
- Browser: Firefox and App
- Memories Version: 7.3.1
- Nextcloud Version: 29.03
- PHP Version: 8.2.20
```

### Screenshots

![Screenshot_20240714_083333_Fennec](https://github.com/user-attachments/assets/6f63b2bb-0f3e-4e49-b76a-8dee2382b26a)

![Screenshot_20240714_083318_Fennec](https://github.com/user-attachments/assets/801ca552-317a-488f-ba8c-3b770ab86fb2)

### Additional context

- Any errors in the JS console? No
- Any errors in the Nextcloud server logs? No

**Comment by pulsejet** [comment]
Yeah, `en_gb` translators decided name = surname for some reason.

**Comment by RikkiBC** [comment]
Hi @pulsejet , apologies I just couldn't tell what you meant by your comment... does that mean this is something that can be fixed within this issue, or should I raise this in a different repo instead?

**Comment by richardwonka** [comment]
Can confirm.
Likely the input field is set to a type/class that in a _person’s_ name (correctly) translates to “surname” (or “family name”, as opposed to “given name”). 

It appears that the input field for the album name is badly classed.

**Comment by pulsejet** [comment]
This needs to be fixed in transifex. The `en_gb` translation for `Name` is currently "Surname"

https://github.com/pulsejet/memories/blob/b5326c47de0e63b3b43075768fc01aa3a663c1f1/l10n/en_GB.json#L271

**Comment by richardwonka** [comment]
> This needs to be fixed in transifex. The `en_gb` translation for `Name` is currently "Surname"
> [...]
That is a correct translation, depending on context. Is the translation in transifex specific to this use case or general? 

I'd suggest defining context also in the original label, e.g. changing the original label to "Album Name" (Title Case optional :-D ) and then following up with translations accordingly.

**Comment by pulsejet** [comment]
> That is a correct translation, depending on context.

I don't agree, there's no context in Memories where "Surname" would be a valid translation.

> I'd suggest defining context also in the original label, e.g. changing the original label to "Album Name" 

Regardless, this seems reasonable (https://github.com/pulsejet/memories/commit/ad54a420a2d64999e1316cfa34c459813c78c063)
